node_exporter.go

```diff
@@ -37,6 +38,9 @@ import (
 	"github.com/prometheus/node_exporter/collector"
 )
 
+// list of enabled collectors used for logging and filtering.
+var enabledCollectors []string
```

Comment:
This can not be a a package variable. This is not thread safe and will cause data races.

Reply:
I tried to create a race condition by concurrently hitting `/metrics` but couldn't trigger it.
Will moving the variable to `handler` structure resolve the potential race issue?

Reply:
Oh, yes, you're right, this is read-only in `ServeHTTP()`. So there is no real race.

But, yes, moving it to the `handler` struct would be cleaner.